lib/pages/feedback.dart

```diff
@@ -1,113 +1,88 @@
-                  color: theme.textTheme.bodyText1.color,
-                ),
-              ),
-              Container(
-                padding: const EdgeInsets.only(top: 16.0),
-                child: Text(
-                  localization.feedbackContent,
-                  style: theme.textTheme.bodyText1,
-                ),
-              ),
-              Container(
-                padding: const EdgeInsets.only(top: 16.0),
-                child: FutureBuilder<void>(
-                  future: _launched,
-                  builder: (BuildContext context, AsyncSnapshot snapshot) {
-                    return Text(snapshot.hasError ? "${snapshot.error}" : "");
-                  },
+    final feedBack = TrufiConfiguration().configurationDrawer.definitionFeedBack;
+    return Scaffold(
+      appBar: AppBar(title: Text(localization.menuFeedback)),
+      body: ListView(
+        children: <Widget>[
+          Container(
+            padding: const EdgeInsets.all(16.0),
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.stretch,
+              children: <Widget>[
+                Text(
+                  localization.feedbackTitle,
+                  style: theme.textTheme.headline6.copyWith(
+                    color: theme.textTheme.bodyText1.color,
+                  ),
                 ),
-              ),
-            ],
-          ),
-        )
-      ],
-    );
-  }
-
-  Widget _buildFloatingActionButton(BuildContext context) {
-    final cfg = TrufiConfiguration();
-    final theme = Theme.of(context);
-    return FloatingActionButton(
-      backgroundColor: theme.primaryColor,
-      onPressed: () {
-        setState(() {
-          final String url = "mailto:${cfg.email.feedback}?subject=Feedback";
-          _launched = _launch(context, url);
-        });
-      },
-      heroTag: null,
-      child: Icon(Icons.email, color: theme.primaryIconTheme.color),
+                Container(
+                  padding: const EdgeInsets.only(top: 16.0),
+                  child: Text(
+                    localization.feedbackContent,
+                    style: theme.textTheme.bodyText1,
+                  ),
+                )
+              ],
+            ),
+          )
+        ],
+      ),
+      drawer: const TrufiDrawer(FeedbackPage.route),
+      floatingActionButton: FloatingActionButton(
+        backgroundColor: theme.primaryColor,
+        onPressed: () async {
+          String url = '';
+          if (feedBack.type == FeedBackType.email) {
+            url = "mailto:${feedBack.body}?subject=Feedback";
+          } else {
+            url = feedBack.body;
+          }
+          if (await canLaunch(url)) {
+            await launch(url);
+          } else {
+            await showDialog(
+              context: context,
+              builder: (BuildContext context) {
+                return AlertDialog(
+                  // TODO translate
```

Comment:
To remove this TODO, I created a PR https://github.com/trufi-association/trufi-core/pull/454